report/upload_report.go

```diff
@@ -0,0 +1,151 @@
+package report
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"time"
+
+	"github.com/yuin/goldmark"
+	"github.com/yuin/goldmark/extension"
+	"github.com/yuin/goldmark/parser"
+	"github.com/yuin/goldmark/renderer"
+	"github.com/yuin/goldmark/renderer/html"
+)
+
+const (
+	YYYYMMDD = "2006-01-02"
+)
+
+// ConfluencePage represents the JSON payload to create a new Confluence page
+type ConfluencePage struct {
```

Comment:
suggest keeping this unexported (lowercase `confluencePage`) since it's only used internally, otherwise it will show up on the package documentation: https://pkg.go.dev/github.com/xornivore/incidentist@v0.0.0-20240311215743-7464697b61c1/report

Reply:
Done